elements/lisk-cryptography/src/keys.ts

```diff
@@ -81,3 +85,93 @@ export const getAddressFromPrivateKey = (privateKey: string): string => {
 
 	return getAddressFromPublicKey(publicKey);
 };
+
+const GENERATOR = [0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3];
+
+const polymod = (uint5Array: number[]): number => {
```

Comment:
can we add this link for reference, otherwise its hard to follow without comments. https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md#creating-checksum